.ci/build_images.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# set -o xtrace
+
+jobfiles=$(find ./.ci/jobs -name "*.job" | sort | awk "NR % ${CIRCLE_NODE_TOTAL} == ${CIRCLE_NODE_INDEX}")
```

Comment:
Mostly to signal developer that this is not part of the "business logic", and is something they don't need to look at unless they really know what they are doing.

I have renamed it to `ci`.